EncounterHandler.js

```diff
@@ -690,16 +690,18 @@ function inject_monster_image(stat) {
 	}
 	if (window.EncounterHandler.combat_body.find(".encounter-details-content-section__content .injected-image").length == 0) {
 		let content = window.EncounterHandler.combat_body.find(".encounter-details-content-section__content");
-		let image = `<img style="width:100%" class="injected-image" src="${stat.data.largeAvatarUrl}"
-			alt="${stat.data.name}" class="monster-image" onerror="this.src='${stat.data.avatarUrl}'";onerror=''></img>`;
+		const image = `<img style="width:100%" class="injected-image" src="${stat.data.largeAvatarUrl}"
+			alt="${stat.data.name}" class="monster-image" onerror="this.src='${stat.data.avatarUrl}';onerror='';"></img>`;
```

Comment:
I'm sorry for all the back and forth on this one, but I just learned that `onerror` is deprecated 🤦 
I think you should use something like `$(image).on("error", function(event) { ... });` instead. That will probably be a lot easier to read anyway.

Reply:
Was a bit fiddly, I couuldn't just do `$(image)[0].src = stat.data.avatarUrl` so just went with removing the previous image and loading a new one

Reply:
ahhh. Makes sense